UI/src/view/point_asset/obstacleForm.js

```diff
@@ -19,7 +19,11 @@
       2: 'Avattava puomi'
     };
 
-    this.renderValueElement = function(asset) {
+    //TODO: investigate obstacle to be created by point asset form
+    var propertyOrdering = ['suggest_box'];
+
+    this.renderValueElement = function(asset, collection, authorizationPolicy) {
```

Comment:
The unused function warning is appearing because there is no direct call for this method in the current asset. However this will be call by it's parent in PointAssetForm